site/content/docs/5.3/components/alerts.md

```diff
@@ -118,7 +118,7 @@ You can see this in action with a live demo:
 <div class="alert alert-warning alert-dismissible fade show" role="alert">
   <span class="alert-icon"><span class="visually-hidden">Warning</span></span>
   <p>Warning notification text goes here.</p>
-  <button type="button" class="btn-close" data-bs-dismiss="alert"><span class="visually-hidden">Close</span></button>
+  <button type="button" class="btn-close" data-bs-dismiss="alert" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close"><span class="visually-hidden">Close</span></button>
```

Comment:
In the StackBlitz environment, the tooltip won't appear without including the JS from the documentation, so `{{< example >}}` must now be `{{< example stackblitz_add_js="true" >}}`

Please double-check this specific thing in all examples. I know that the next "Dark variant" is impacted as well, but maybe the other files too: modal page, navbar page, etc.